lib/task-data/base-tasks/generate-tag.js

```diff
@@ -0,0 +1,13 @@
+// Copyright 2016, EMC, Inc.
+
+'use strict';
+
+module.exports = {
+    friendlyName: 'Generate Tag',
+    injectableName: 'Task.Base.Catalog.GenerateTag',
```

Comment:
GenerateTag should not be a standard catalog job (For me, Standard Catalog Job means executing catalog command, parse the output and store it to database), so I am wondering whether it is suitable to set the injectableName to be nested under `Catalog`.

@RackHD/corecommitters may have other thought on this.